src/main/java/com/neprozorro/rest/LotInfoResource.java

```diff
@@ -0,0 +1,34 @@
+package com.neprozorro.rest;
+
+import com.neprozorro.model.LotInfo;
+import com.neprozorro.rest.dto.LotInfoCriteriaRequestDto;
+import com.neprozorro.rest.dto.LotInfoMapper;
+import com.neprozorro.rest.dto.LotInfoResponseDTO;
+import com.neprozorro.service.LotInfoService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/lot-info")
+@RequiredArgsConstructor
+public class LotInfoResource {
+
+    private final LotInfoService lotInfoService;
+    private final LotInfoMapper lotInfoMapper;
+
+    @GetMapping
+    public List<LotInfo> findAll() {
+        return lotInfoService.findAll();
+    }
+
+    @GetMapping("/filter")
```

Comment:
I am not sure that two get requests will be to work because I had a problem with them.

Reply:
You have two different endpoints:
/lot-info
/lot-info/filter
I don't see a hypothetical issue. It looks like a regular using @GetMapping with different roads.

Reply:
yes, in case paths' are different it will work fine